graylog2-web-interface/src/pages/ConfigurationsPage.tsx

```diff
@@ -15,187 +15,163 @@
-        )}
-        {isPermitted(currentUser.permissions, ['urlwhitelist:read']) && urlWhiteListConfig && (
-          <ConfigletContainer title="URL Whitelist Configuration">
-            <UrlWhiteListConfig config={urlWhiteListConfig}
-                                updateConfig={_onUpdate(URL_WHITELIST_CONFIG)} />
-          </ConfigletContainer>
-        )}
-        <ConfigletContainer title="Decorators Configuration">
-          <DecoratorsConfig />
-        </ConfigletContainer>
-        {permissionsConfig && (
-          <ConfigletContainer title="Permissions Configuration">
-            <PermissionsConfig config={permissionsConfig}
-                               updateConfig={_onUpdate(PERMISSIONS_CONFIG)} />
-          </ConfigletContainer>
-        )}
-        {indexSetsDefaultsConfig && (
-          <HideOnCloud>
-            <ConfigletContainer title="Index Set Default Configuration">
-              <IndexSetsDefaultsConfig initialConfig={indexSetsDefaultsConfig}
-                                       updateConfig={_onUpdate(INDEX_SETS_DEFAULTS_CONFIG)} />
-            </ConfigletContainer>
-          </HideOnCloud>
-        )}
-        {userConfig && (
-          <ConfigletContainer title="User Configuration">
-            <UserConfig config={userConfig}
-                        updateConfig={_onUpdate(USER_CONFIG)} />
-          </ConfigletContainer>
-        )}
-      </>
-    );
-  }
-
   return (
     <DocumentTitle title="Configurations">
-      <span>
-        <PageHeader title="Configurations">
-          <span>
-            You can configure system settings for different sub systems on this page.
-          </span>
-        </PageHeader>
-
-        <ConfigletRow className="content">
-          {Output}
-        </ConfigletRow>
-
-        {pluginSystemConfigs.length > 0 && (
-        <Row className="content">
-          <Col md={12}>
-            <h2>Plugins</h2>
-            <p className="description">Configuration for installed plugins.</p>
-            <hr className="separator" />
-            <div className="top-margin">
-              <PluginConfigRows configuration={configuration} systemConfigs={pluginSystemConfigs} />
-            </div>
-          </Col>
-        </Row>
-        )}
-      </span>
+      <PageHeader title="Configurations">
+        <span>
+          You can configure system settings for different sub systems on this page.
+        </span>
+      </PageHeader>
+
+      <ConfigletRow className="content">
+        <Col md={2}>
+          <Nav bsStyle="pills" stacked activeKey={activeSectionKey} onSelect={handleNavSelect}>
+            {configurationSections.map(({ hide, name, showCaret }, index) => (
+              !hide && (
+              <NavItem key={`nav-${name}`} eventKey={index + 1} title={name}>
+                {name}
+                {showCaret && (isSectionActive(name)
+                  ? <Icon name="caret-right" style={{ marginLeft: 5, verticalAlign: 'middle' }} />
```

Comment:
Since we agreed on avoiding inline styling, can we convert these to styled components?